lib/tasks/cssbundling/build.rake

```diff
@@ -21,42 +21,42 @@ module Cssbundling
   module Tasks
     extend self
 
+    LOCK_FILES = {
+      bun: %w[bun.lockb bun.lock yarn.lock],
```

Comment:
When you have bun installed on the system, but you are building the project with yarn and having a yarn.lock, i think we should remove the File.exist?('yarn.lock') even if bun manage yarn.lock, to avoid the confusion.